server/lib/redis/index.ts

```diff
@@ -1,8 +1,20 @@
 import Redis from 'ioredis'
+import logger from '../logging/logger'
 
 // TODO(tec27): provide some better wrapper around this that deals with connects/disconnects, etc.
-export default new Redis({
+const redis = new Redis({
   port: Number(process.env.SB_REDIS_PORT),
   host: process.env.SB_REDIS_HOST,
   dropBufferSupport: true,
 })
+
+redis.on('error', err => {
+  logger.error(err)
```

Comment:
So the way the logger works is that the first parameter passed to it is an object where the keys can have special meaning. Depending on the key it filters out certain properties to log as well, the ones that have special handling are:

 - `err` (for `Error`s)
 - `req` (for `HttpRequest`s)
 - `res` (for `HttpResponse`s)

so it's important that when you're logging things with it, you pass an object that has the right keys. It's also important that you include some message so we have a better idea of what is occurring

So a better call here is probably something like:
```ts
logger.error({ err }, 'redis error')
```